<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Event::services::ServiceWidget</class>
 <widget class="QWidget" name="Event::services::ServiceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QLabel" name="lblStatus">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../../../../libqf/libqfcore/images/images.qrc">:/qf/core/images/light-red.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblServiceName">
     <property name="text">
      <string>EmmaClient</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblServiceMessage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>neco neco</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btPlay">
     <property name="icon">
      <iconset resource="../../../../../../libqf/libqfqmlwidgets/images/flat/qfqmlwidgets_flat.qrc">
       <normaloff>:/qf/qmlwidgets/images/flat/media-play.svg</normaloff>:/qf/qmlwidgets/images/flat/media-play.svg</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btShowDetail">
     <property name="icon">
      <iconset resource="../../../../../../libqf/libqfqmlwidgets/images/flat/qfqmlwidgets_flat.qrc">
       <normaloff>:/qf/qmlwidgets/images/flat/settings.svg</normaloff>:/qf/qmlwidgets/images/flat/settings.svg</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../../libqf/libqfcore/images/images.qrc"/>
  <include location="../../../../../../libqf/libqfqmlwidgets/images/flat/qfqmlwidgets_flat.qrc"/>
 </resources>
 <connections/>
</ui>
